### DISCUSSION TITLE
Persistent tag views

### DISCUSSION THREAD

**Discussion by wsw70** [usememos/memos#3809] [Ideas]
It would be awesome to be able to have persistent, reusable filters

![image](https://github.com/user-attachments/assets/b473b40f-7c71-48b5-bd17-c269f7cee011)

**Comment by RoccoSmit** [comment]
The feature will be in the next release. Commit reference: https://github.com/usememos/memos/pull/4239

**Comment by RoccoSmit** [comment]
its in v0.23.1
